lib/src/kvs/tests/lq.rs

```diff
@@ -37,3 +43,556 @@ async fn scan_node_lq() {
+			}
+		}
+		_ => panic!("Expected a uuid"),
+	};
+	println!("Created entry");
+
+	// Register a live query on the remote node
+	let tx = test.db.transaction(Write, Optimistic).await.unwrap().enclose();
+	let live_value =
+		compute_live(&ctx, &remote_options, tx.clone(), live_query_id, remote_node, table).await;
+	tx.lock().await.commit().await.unwrap();
+	assert_eq!(live_value, Value::Uuid(sql::uuid::Uuid::from(live_query_id)));
+	println!("Created live query in test");
+
+	// Write locally to cause a remote notification
+	let thing = match create_value.get("id").unwrap().clone() {
+		Value::Thing(thing) => thing,
+		_ => panic!("Expected ID to be a thing"),
+	};
+	let tx = test.db.transaction(Write, Optimistic).await.unwrap().enclose();
+	let delete_value = compute_delete(&ctx, &local_options, tx.clone(), thing.clone()).await;
+	tx.lock().await.commit().await.unwrap();
+	// Delete returns empty
+	assert_eq!(Value::Array(Array::new()), delete_value);
+
+	// Verify local node did not get notification
+	assert!(test.db.notifications().unwrap().try_recv().is_err());
+
+	// Verify there is a remote node notification entry
+	let tx = test.db.transaction(Write, Optimistic).await.unwrap().enclose();
+	let mut res = tx
+		.lock()
+		.await
+		.scan_tbnt(
+			namespace.as_str(),
+			database.as_str(),
+			table,
+			sql::uuid::Uuid::from(live_query_id),
+			1000,
+		)
+		.await
+		.unwrap();
+	tx.lock().await.commit().await.unwrap();
+
+	// Validate there is a remote notification
+	assert_eq!(res.len(), 1);
+	let not = res.get_mut(0).unwrap();
+	// Notification ID is random, so we set it to a known value
+	assert!(!not.notification_id.is_nil());
+	not.notification_id = Default::default();
+	// The notification value for delete is just the ID of the record
+	let expected_remote_notification = Notification {
+		live_id: crate::sql::uuid::Uuid::from(live_query_id),
+		node_id: crate::sql::uuid::Uuid::from(remote_node),
+		notification_id: Default::default(),
+		action: Action::Delete,
+		result: Value::Thing(thing),
+		timestamp: t1,
+	};
+	assert_eq!(not, &expected_remote_notification);
+}
+
+async fn compute_live<'a>(
+	ctx: &'a context::Context<'a>,
+	opt: &'a Options,
+	tx: Arc<Mutex<Transaction>>,
+	live_query_id: Uuid,
+	node_id: Uuid,
+	table: &'a str,
+) -> Value {
+	let live_stm = LiveStatement {
+		id: sql::uuid::Uuid::from(live_query_id),
+		node: sql::uuid::Uuid::from(node_id),
+		expr: Fields(vec![sql::Field::All], false),
+		what: Value::Table(sql::table::Table::from(table.to_owned())),
+		cond: None,
+		fetch: None,
+		archived: None,
+		session: Some(Value::None),
+		auth: None,
+	};
+	live_stm.compute(ctx, opt, &tx, None).await.unwrap()
+}
+
+async fn compute_create<'a>(
+	ctx: &'a context::Context<'a>,
+	opt: &'a Options,
+	tx: Arc<Mutex<Transaction>>,
+	table: &'a str,
+	what: Option<Thing>,
+) -> Value {
+	let mut map: BTreeMap<String, Value> = BTreeMap::new();
+	map.insert("name".to_string(), Value::Strand(Strand::from("a name")));
```

Comment:
Is having the value of `name` be "a name" here intentional? Do we need it? If we need `data` to be set to something, can't we use `Data:: EmptyExpression` here?

Reply:
Never mind. I forgot this is in a `tests` module.